database/seeders/DatabaseSeeder.php

```diff
@@ -15,11 +18,10 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        UserFactory::new()->count(10)->create();
+        CityFactory::new()->count(40)->create();
+
+        AirlineFactory::new()->count(15)->create();
```

Comment:
```suggestion
        CityFactory::times(40)->create();

        AirlineFactory::times(15)->create();
```
Same thing, less code.